benchmark/run.jl

```diff
@@ -0,0 +1,9 @@
+using Pkg
+Pkg.develop(PackageSpec(path=pwd()))
```

Comment:
`pwd()` can differ depending on where the user has invoked Julia, `cd`'d to, etc. It's a good option when the code is written inline in the YAML file but is a bit iffy to have in a file. To ensure you're always pointing at the right thing, I'd make this:
```suggestion
Pkg.develop(PackageSpec(path=dirname(@__DIR__)))
```